mlflow/protos/service.proto

```diff
@@ -707,6 +719,42 @@ message ListExperiments {
   }
 }
 
+message SearchExperiments {
+  option (scalapb.message).extends = "com.databricks.rpc.RPC[$this.Response]";
+  option (scalapb.message).extends = "com.databricks.mlflow.api.MlflowTrackingMessage";
+
+  // Maximum number of experiments desired.
+  // Servers may select a desired default `max_results` value. All servers are
+  // guaranteed to support a `max_results` threshold of at least 1,000 but may
+  // support more. Callers of this endpoint are encouraged to pass max_results
+  // explicitly and leverage page_token to iterate through experiments.
+  optional int64 max_results = 1;
+
+  // Token indicating the page of experiments to fetch
+  optional string page_token = 2;
+
+  // String representing a SQL filter condition (e.g. "name ILIKE 'my-experiment%'")
+  optional string filter = 3;
+
+  // List of columns for ordering search results, which can include experiment name and last updated
```

Comment:
The ordering column list can also include experiment ID, right?

Reply:
Yes, fixed.